docs/data-sources/container_registry.md

```diff
@@ -0,0 +1,68 @@
+---
+subcategory: "Container Registry"
+layout: "ionoscloud"
+page_title: "IonosCloud : ionoscloud_container_registry"
+sidebar_current: "docs-ionoscloud_container_registry"
+description: |-
+Get information on a Container Registry
+---
+
+# ionoscloud_container_registry
+
+⚠️ **Note:** Container Registry is currently in the Early Access (EA) phase. We recommend keeping usage and testing to non-production critical applications.
+Please contact your sales representative or support for more information.
+
+The **Container Registry data source** can be used to search for and return an existing Container Registry.
+You can provide a string for the name parameter which will be compared with provisioned Container Registry.
+If a single match is found, it will be returned. If your search results in multiple matches, an error will be returned.
+When this happens, please refine your search and make sure that your resources have unique names.
+
+## Example Usage
+
+### By Id
+```hcl
+data "ionoscloud_container_registry" "example" {
+  id			= <registry_id>
+}
+```
+
+### By Name
+```hcl
+data "ionoscloud_container_registry" "example" {
+  name			= "container-registry-example"
+}
+```
+
+### By Name with Partial Match
+```hcl
+data "ionoscloud_container_registry" "example" {
+  name			= "-example"
+  partial_match = true
+}
+```
+
+## Argument Reference
+
+* `id` - (Optional) ID of the container registry you want to search for.
+* `name` - (Optional) Name of an existing container registry that you want to search for. Search by name is case-insensitive. The whole resource name is required if `partial_match` parameter is not set to true.
+* `partial_match` - (Optional) Whether partial matching is allowed or not when using name argument. Default value is false.
+
+Either `name` or `id` must be provided. If none, or both of `name` and `id` are provided, the datasource will return an error.
+
+## Attributes Reference
+
+The following attributes are returned by the datasource:
+
+* `id` - Id of the container registry.
```

Comment:
@cristiGuranIonos in my opinion, the descriptions for these attributes are useless since the role of the attribute can easily be determined by reading the name. However, I noticed that in other files, for example `k8s_cluster.md` we are adding a description for each attribute. I suggest removing these descriptions for the current file and maybe only add descriptions where it's the case, maybe when the attribute name is not clear enough.